packages/microservice/src/bus-adapters/redis-bus.adapter.ts

```diff
@@ -0,0 +1,86 @@
+import { Logger } from '@nestjs/common';
+import { IBusAdapter } from '../interfaces/bus/bus-adapter.interface';
+import { ITransferData } from '../interfaces/transfer-data';
+import { TransferDataDto } from '../interfaces/transfer-data-dto.interface';
+import { loadPackage } from '../utils/load-package.util';
+import { IOnInit } from '../interfaces/lifecycles';
+import { ISetOptions } from '../interfaces/set-options.interface';
+
+let redisPackage: any = {};
+
+export class RedisBusAdapter implements IBusAdapter, IOnInit, ISetOptions {
+
+    private pub;
+
+    private sub;
+
+    private options = {};
+
+    private handles: Map<string, Function[]>;
+
+    private readonly logger: Logger;
+
```

Comment:
Take care with unnecessary tabulations, (not worry this happens to me too)

I think we need our custom strong `tslint` for this.